spec/controllers/collection_participants_controller_spec.rb

```diff
@@ -387,7 +392,8 @@
 
         it "approves those users, redirects to the collection participants page and displays a notification" do
           get :add, params
-          it_redirects_to collection_participants_path(collection)
+          expect(response).to have_http_status :redirect
```

Comment:
That is because we are not certain how the users will be added to the notice method.